nbbuild/antsrc/org/netbeans/nbbuild/extlibs/ignored-overlaps

```diff
@@ -34,6 +34,9 @@ ide/o.apache.commons.lang/external/commons-lang-2.6.jar java/maven.embedder/exte
 # upstream problem
 nbbuild/external/langtools-9.zip nbbuild/external/langtools-9.zip
 
+# Used to parse data during build, but 
+nbbuild/external/json-simple-1.1.1.jar
```

Comment:
```
# List of patterns (one pair per line) to skip over in VerifyLibsAndLicenses.testBinaryUniqueness.
# Both sides of the pair must match for the pair to be skipped.
```
Think you need the other file location here too?!  Or is it working locally anyway?